src/templates/_common/brand.config.js

```diff
@@ -0,0 +1,38 @@
+const brand_config_data = {
+    brand_name: 'Deriv',
+    platforms : {
+        smarttrader: {
+            name: 'SmartTrader',
+            icon: 'logo_smart_trader.svg',
+        },
+        dtrader: {
+            name: 'DTrader',
+            icon: 'ic-brand-dtrader.svg',
+        },
+        dbot: {
+            name: 'DBot',
+            icon: 'ic-brand-dbot.svg',
+        },
+        dmt5: {
+            name     : 'DMT5',
+            full_name: 'Deriv MetaTrader 5',
+            icon     : 'ic-brand-dmt5.svg',
+        },
+        dxtrade: {
+            name: 'Deriv X',
+            icon: 'ic-brand-dxtrade.svg',
+        },
+        binarybot: {
```

Comment:
I would advise not to use our brand name as a key inside the config. It makes less sense if someone decides to fork our repo and still use `binarybot` as a key, so we better make it as generic as possible

Reply:
I've renamed it to bbot 👌🏻

Reply:
@matin-binary just now 😊